apps/proxy/src/app/users/user/user.component.html

```diff
@@ -0,0 +1,100 @@
+            ></date-range-picker>
+            <button
+                mat-flat-button
+                class="flat-icon-btn flat-default"
+                (click)="getUsers()"
+                matTooltip="Refresh"
+                matTooltipPosition="above"
+            >
+                <mat-icon class="mat-icon-18">refresh</mat-icon>
+            </button>
+        </div>
+    </div>
+    <mat-card class="p-0 outline-card responsive-card flex-grow-1">
+        <mat-card-content>
+            <div class="table-scroll">
+                <table
+                    mat-table
+                    [dataSource]="
+                        (users$ | async)?.data?.length
+                            ? (users$ | async)?.data
+                            : (isLoading$ | async)
+                            ? [1, 2, 3, 4]
+                            : []
+                    "
+                    class="default-table responsive-table"
+                >
+                    <!-- Name Column -->
+                    <ng-container matColumnDef="name">
+                        <th mat-header-cell *matHeaderCellDef mat-sort-header class="width-md-300">Name</th>
+                        <td mat-cell *matCellDef="let element" data-label="Name">
+                            <ng-container *ngIf="!(isLoading$ | async); else loading">
+                                {{ element.name }}
+                            </ng-container>
+                        </td>
+                    </ng-container>
+
+                    <!-- Email Column -->
+                    <ng-container matColumnDef="email">
+                        <th mat-header-cell *matHeaderCellDef mat-sort-header class="width-md-300">Email</th>
+                        <td mat-cell *matCellDef="let element" data-label="Email">
+                            <ng-container *ngIf="!(isLoading$ | async); else loading">
+                                {{ element.email }}
+                            </ng-container>
+                        </td>
+                    </ng-container>
+
+                    <!-- Phone Number Column -->
+                    <ng-container matColumnDef="phone_number">
+                        <th mat-header-cell *matHeaderCellDef mat-sort-header>Phone Number</th>
+                        <td mat-cell *matCellDef="let element" data-label="Phone Number">
+                            <ng-container *ngIf="!(isLoading$ | async); else loading">
+                                {{ element.mobile ?? '-' }}
+                            </ng-container>
+                        </td>
+                    </ng-container>
+
+                    <tr mat-header-row *matHeaderRowDef="displayedColumns; sticky: true"></tr>
+                    <tr
+                        mat-row
+                        *matRowDef="let row; columns: displayedColumns; let i = index"
+                        [ngClass]="{ 'last-child': i + 1 === pageSizeOptions[0] }"
```

Comment:
Avoid using hard coded index in `pageSizeOptions`

Reply:
done